README.md

```diff
@@ -33,28 +33,29 @@ In case you want to train your own model to use with Eynollah, have a look at [s
 The command-line interface can be called like this:
 
 ```sh
-eynollah \
--i <image file name> \
--o <directory to write output xml or enhanced image> \
--m <directory of models> \
--fl <if true, the tool will perform full layout analysis> \
--ae <if true, the tool will resize and enhance the image and produce the resulting image as output. The rescaled and enhanced image will be saved in output directory> \
--as <if true, the tool will check whether the document needs rescaling or not> \
--cl <if true, the tool will extract the contours of curved textlines instead of rectangle bounding boxes> \
--si <if a directory is given here, the tool will output image regions inside documents there> \
--sd <if a directory is given, deskewed image will be saved there> \
--sa <if a directory is given, all plots needed for documentation will be saved there> \
--tab <if true, this tool will try to detect tables> \
--ib <in general, eynollah uses RGB as input but if the input document is strongly dark, bright or for any other reason you can turn binarized input on. This option does not mean that you have to provide a binary image, otherwise this means that the tool itself will binarized the RGB input document> \
--ho <if true, this tool would ignore headers role in reading order detection> \
--sl <if a directory is given, plot of layout will be saved there> \
--ep <if true, the tool will be enabled to save desired plot. This should be true alongside with -sl, -sd, -sa , -si or -ae options>
--light <if true, the tool will apply a faster method for main regions detection and moreover deskewing will not occur locally for each region in return it is done once for the whole document>
--di <directory of images. This option will accelerate the process significantly>
+eynollah -i <image file name> -o <directory to write output> -m <directory of models> [OPTIONS]
+```
+
+Additionally, the following optional parameters can be used to further configure the processing:
 
+```sh
+-fl: the tool will perform full layout analysis including detection of marginalia and drop capitals 
```

Comment:
full layout detection is responsible for header and drop capitals not marginalia.